### PR TITLE
Bug 1550858 - Update final final final final final actions or urls

### PR DESCRIPTION
**Pull request by Mardak**
r?@rlr or @k88hudson I suppose I could have used asrouter targeting `localeLanguageCode` but wasn't sure about duplicating `id` or having a whole separate action and other interactions…

### CODE REVIEW

**Comment by Mardak** [comment]
Oh, hrm. I guess it does say `(Redirected from Send Tab from Firefox desktop to mobile) ` .. sometimes

File: lib/OnboardingMessageProvider.jsm [hunk 1]
```diff
@@ -223,7 +224,10 @@ const ONBOARDING_MESSAGES = async () => ([
       icon: "tracking",
       primary_button: {
         label: {string_id: "onboarding-tracking-protection-button2"},
-        action: {
+        action: Services.locale.appLocaleAsLangTag.substr(0, 2) === "en" ? {
+          type: "OPEN_URL",
+          data: {args: "https://mzl.la/ETPdefault", where: "tabshifted"},
```
Comment (rlr) [inline lib/OnboardingMessageProvider.jsm]: I guess this will be published that day.

Comment (k88hudson) [reply lib/OnboardingMessageProvider.jsm]: Is there some documentation for this URL somewhere?

Comment (Mardak) [reply lib/OnboardingMessageProvider.jsm]: https://bugzilla.mozilla.org/show_bug.cgi?id=1550858#c7

**Pull request merged**